public/app/core/services/context_srv.ts

```diff
@@ -279,7 +279,7 @@ export class ContextSrv {
       });
   }
 
-  private getSessionExpiry() {
+  getSessionExpiry() {
```

Comment:
This function does not have to be a member function, maybe we should just make it a stand alone function and use it in both context_srv and backend_srv, WDYT?